### PR TITLE
chore: extract @k8slens/animate

### PR DESCRIPTION
**Pull request by gabriel-mirantis**

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

**Review by Nokel81 — APPROVED** [review]

File: packages/ui-components/animate/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```bash
+$ npm install @k8slens/animate
+```
+
+```typescript
+export type { AnimateName, AnimateProps } from "@k8slens/button";
+export type { Animate } from "@k8slens/button";
+export { defaultEnterDurationForAnimatedInjectable } from "@k8slens/button";
+export { defaultLeaveDurationForAnimatedInjectable } from "@k8slens/button";
+export { requestAnimationFrameInjectable } from "@k8slens/button";
```
Comment (jansav) [inline packages/ui-components/animate/README.md:16]: `@k8slens/button` -> `@k8slens/animate`

Comment (jansav) [reply packages/ui-components/animate/README.md:16]: Also prefer exporting `injectionToken` and not `injectable` from the feature.

Comment (Nokel81) [reply packages/ui-components/animate/README.md:16]: @jansav We haven't made a full decision on preferring `injectionTokens` over `injectables`, plus it adds more work that isn't required ATM

Comment (jansav) [reply packages/ui-components/animate/README.md:16]: Exporting `injectable` introduces so many blindspots that we just shouldn't do it. It has been the instruction from the beginning. It's even listed in the [README.md](https://www.npmjs.com/package/@k8slens/feature-core)

Eg. blindspots:
- What happens when other feature registers the injectable?
- What happens when other feature deregisters the injectable?

Comment (Nokel81) [reply packages/ui-components/animate/README.md:16]: @Iku-turso Doesn't completely agree as I understand. And both of those of those blindspots could easily be blocked in code instead, which would be even better.

**Pull request merged**